cdap-ui/app/cdap/components/Experiments/DetailedView/ModelsTable/index.js

```diff
@@ -118,153 +100,166 @@ const getNewHeadersBasedOnOutcome = (outcomeType) => (
     addMetricsToHeaders(tableHeaders, categoricalMetrics)
 );
 
-const renderTableHeaders = (outcomeType, renderSortableTableHeader) => {
+const addDetailedModelObject = (list) => {
+  let activeIndex = list.findIndex(model => model.active);
+  if (activeIndex !== -1) {
+    return [
+      ...list.slice(0, activeIndex + 1),
+      {
+        ...list[activeIndex],
+        detailedView: true,
+        active: false
+      },
+      ...list.slice(activeIndex + 1)
+    ];
+  }
+  return list;
+};
+
+const wrapContentWithTitleAttr = (content) => (
+  <div title={isNumber(content) || isString(content) ? content : ''}>
+    {content}
+  </div>
+);
+
+const renderMetrics = (newHeaders, model) => {
+  let commonHeadersLen = tableHeaders.length;
+  let len = newHeaders.length;
+  let metrics = newHeaders.slice(commonHeadersLen, len);
+  return metrics.map(t => wrapContentWithTitleAttr(model.evaluationMetrics[t.property] || '--'));
+};
+
+const renderModelDetails = (model) => {
+  let Component = 'div';
+  let props = {
+    className: classnames("grid-item", {
+      "opened": model.detailedView,
+      "active": model.active
+    }),
+    key: shortid.generate()
+  };
+  return (
+    <Component {...props}>
```

Comment:
If you're just setting `Component = 'div'` and then not change `Component`, why not just use `div` here right away?